__tests__/app.test.js

```diff
@@ -200,3 +197,47 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles?sort_by=votes&order=asc", () => {
+  test("gives status of 200 and responds with all articles sorted by votes and ordered by ascending", () => {
+    return request(app)
+      .get("/api/articles?sort_by=votes&order=asc")
+      .expect(200)
+      .then(({ body: { allArticles } }) => {
+        console.log(allArticles)
+        allArticles.forEach((article) => {
+          expect(Object.keys(article)).toHaveLength(8);
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+        });
+
+        expect(allArticles).toBeSortedBy("votes", { ascending: true });
+      });
+  });
+
+  test("gives status of 400 and responds with bad request when passed nonsensical sort_by", () => {
+    return request(app)
+      .get("/api/articles?sort_by=DROPTABLE&order=asc")
+      .expect(404).then(({body: {msg}})=> {
+        expect(msg).toBe('sort_by Not Found')
+       
+      })
+  });
+
+  test("gives status of 400 and responds with bad request when passed nonsensical sort_by", () => {
+    return request(app)
+      .get("/api/articles?sort_by=votes&order=DROPTABLE")
+      .expect(404).then(({body: {msg}})=> {
+        expect(msg).toBe('order Not Found')
+       
+      })
```

Comment:
The status code for invalid queries should be 400 (Bad Request), but your tests expects 404.